lib/task_details_page.dart

```diff
@@ -19,6 +19,7 @@ class TaskDetailsPage extends StatefulWidget {
   _TaskDetailsPageState createState() => _TaskDetailsPageState();
 }
 
+// Carlo
 class _TaskDetailsPageState extends State<TaskDetailsPage> {
   late TextEditingController _titleController;
```

Comment:
Here you could initialise directly. E.g.: late TextEditingController _titleController = TextEditingController();